### ISSUE TITLE
Stub files analyzing

### ISSUE THREAD

**Issue by isacjunior** [BetterThanTomorrow/calva#2465]
When a project is opened, the Calva is watching the stubs files `path/.lsp/.cache/stubs` and sending it to clojure-lsp to analyze and problems are shown on terminal.  Interestingly, if the stub file noisy is opened and later close, the warning from VS Code fade away.

![image](https://github.com/BetterThanTomorrow/calva/assets/12630335/754c1e54-63a1-4972-be37-bb49d2169e67)

**Comment by bpringe** [comment]
Thanks for the issue.

> Calva is watching the stubs files path/.lsp/.cache/stubs and sending it to clojure-lsp to analyze and problems are shown on terminal.

Can you clarify what you mean by "problems are shown on terminal"? Do you mean in the problems pane, like below?
<img width="109" alt="image" src="https://github.com/BetterThanTomorrow/calva/assets/12722744/4b051b21-b3f9-4667-8225-3a85740b69f0">

**Comment by bpringe** [comment]
Adding context here from Slack from @ericdallo.

> The clojure-lsp stub is a clojure-lsp feature which uses [clj-easy/stub](https://github.com/clj-easy/stub) lib to generate dummy clojure files to have clj-kondo analysis, so for example one would get auto complete for datomic.api for example, a lib which is closed source

> This feature spawns under the hood a clj process which exports the ns and vars, it's only enalbed if one has a :stub setting in clojure-lsp config

Slack thread: https://clojurians.slack.com/archives/CBE668G4R/p1676552690524579

**Comment by isacjunior** [comment]
> Do you mean in the problems pane, like below?

Yes
![Screenshot 2024-03-26 at 09 40 13](https://github.com/BetterThanTomorrow/calva/assets/12630335/4efc081c-f463-4e32-b3db-9537eba35313)

**Comment by ericdallo** [comment]
@isacjunior do you opened that api.clj file at any moment? since that folder is not in the classpath, I would expect clojure-lsp to only report diagnostics if you manually open that file 🤔

**Comment by isacjunior** [comment]
@ericdallo no, I do not open that file. Just opening the VSCode and the problems in the terminal is shown.
